src/plugins/intel_gna/src/backend/gna_limitations.cpp

```diff
@@ -395,6 +190,66 @@ std::string RectLimitByChannelsAndPrecision::GetErrorOrEmpty(const uint32_t h,
     return GetByPrecision(precision).GetErrorOrEmpty(h, w, channels, what);
 }
 
+class Validator_30 : public AbstractValidator {
+    static const RangeLimit2D kInputHWLimit;
```

Comment:
I would put private at the end of declaration order.

Reply:
AbstractValidator class is out of refactoring scope for this PR, I focused only on Limitations class